flo_ai/router/flo_llm_router.py

```diff
@@ -72,13 +72,15 @@ def __init__(
             flo_team: FloTeam,
             router_prompt: str = None,
             llm: Union[BaseLanguageModel, None] = None,
+            model_nick_name: str = 'default',
```

Comment:
There is a create method above, we should pass the model name from there to the Builder too

Reply:
Check the `create` method in supervisor